### PR TITLE
Fixing bug

### PR DESCRIPTION
**Pull request by abhiahirv1**

### CODE REVIEW

File: src/main/controllers/features/sole-trader/sectorYouWorkInController.ts [hunk 1]
```diff
@@ -76,16 +76,15 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
             if (req.body.sectorYouWorkIn === "OTHER") {
                 res.redirect(addLangToUrl(BASE_URL + SOLE_TRADER_WHICH_SECTOR_OTHER, lang));
             } else {
-                if (acspData) {
-                    acspData.workSector = req.body.sectorYouWorkIn;
-                }
                 //  save data to mongodb
+                acspData.workSector = req.body.sectorYouWorkIn;
```
Comment (sampankumar) [inline src/main/controllers/features/sole-trader/sectorYouWorkInController.ts:80]: So does this mean acspData can never be null?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/controllers/features/sole-trader/sectorYouWorkInController.ts (hunk 1 above)
Comment (abhiahirv1) [reply src/main/controllers/features/sole-trader/sectorYouWorkInController.ts:80]: technically yes but putting back how it was as that is more secure and better option.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Pull request merged**